README.md

```diff
@@ -807,15 +807,64 @@ When you have time, we invite you to take a look at:
  + [The dynamic router EIP](https://camel.apache.org/components/latest/eips/dynamicRouter-eip.html)
  + [The kamelet EIP](https://camel.apache.org/components/latest/eips/kamelet-eip.html)
 
-## Part 7 - Camel Quarkus and Kafka
+## Part 7 - Integration Example with Camel Quarkus
 Estimate time : 25 minutes
-@TODO: 25 MINUTES LEFT at this level
+
+### About this section
+In this part, we will discover how easy and fast it is to :
+* Stream data from and to Kafka
+* Write to Slack
+* Read / write to an SQL database with JPA
+* Use external APIs
+* Serialize Data
+* Create REST API
+
+
+We will also experience the joy of using [dev services](https://quarkus.io/guides/dev-services) in the development mode. No need to install any kafka server or database.
+
+---
+**🚀NOTE**
+
+This section requires having Docker installed in order to provition a Kafka and Postgresql servers. 
+
+In case, you don't have a proper Docker license:
+* Use H2 Databse instead of Postgresql
+* Ask us during the session to provide you a kafka instance in the Cloud.
+
+---
+
+### Introducing the example
+
+This example will illustrate a application that will receive and process coffee orders. The application is designed around Event Driven Architecture and uses Kafka to communicate with external systems. The orders are stored withing a database. The delivery team is notified on slack to go check the order. The delivery team can access information about orders to deliver using an API.
+
+
+* insert the orders within a Database
+* notify the delivery company using a specific API (This one will be provided during the session)
+
+The application is designed around Event Driven Architecture and uses Kafka to handle those messages:
+* The beer orders to be processed are in a kafka topic named `orders`
```

Comment:
Is there beer amongst with coffee orders too ? Or is it just coffee ?

Reply:
just coffee I changed my mind